.editorconfig

```diff
@@ -0,0 +1,20 @@
+# top-most EditorConfig file
```

Comment:
can we take some from https://github.com/aurae-runtime/ae/blob/main/.editorconfig ?

Reply:
done